src/components/Gyoithon/Gyoithon.tsx

```diff
@@ -18,28 +24,42 @@ interface FormValues {
     export_path: string;
 }
 
-const title = "AI-based pen-testing tool (Gyoithon)";
-const description_userguide =
+// Component Constants
+const title = "Gyoithon"; // Contains the title of the component.
+const sourceLink = "https://github.com/gyoisamurai/GyoiThon"; // Contains the link to the source code.
+const description_userguide = // Contains the description of the component.
```

Comment:
Change this to description - description_userguide is deprecated. 
```suggestion
const description = // Contains the description of the component.
```